scripts/apidev.py

```diff
@@ -1,24 +1,25 @@
+import copy
 import json
 import os
 from http.server import (
     HTTPServer,
     SimpleHTTPRequestHandler,
 )
 
+import requests
 from watchdog.events import FileSystemEventHandler
 from watchdog.observers import Observer
 
 from azul import config
-from azul.deployment import aws
 from azul.modules import load_app_module
 from azul.openapi import annotated_specs
 
 spec_file = 'openapi.json'
 parent_dir = os.path.realpath(os.path.dirname(__file__))
 
 
-def write_specs(gateway_id, app, openapi_spec):
-    specs = annotated_specs(gateway_id, app, openapi_spec)
+def write_specs(raw_specs, app, openapi_spec):
+    specs = annotated_specs(copy.copy(raw_specs), app, openapi_spec)
```

Comment:
This seems like a unrelated change or fix to an unrelated issue. Please elaborate here or in a code comment as to why A) a copy is being made and B) the copy is only shallow?

Reply:
I made this change because I was concerned that it would prevent the `apidev.py` script from displaying the correct specifications in some cases. After looking into this more deeply, I don't believe this would be the case.

If it was necessary however, it should have been a deep copy.

Reply:
There is a separate issue here however, which I will address in a separate commit

Reply:
On further reflection, I don't believe there's an issue here.